### PR TITLE
Release v1.4.5

### PR DESCRIPTION
**Pull request by philpl**

### CODE REVIEW

**Comment by ismay** [comment]
Since this release concerns itself with fixing the prop-types warnings, does this not also need this pr: https://github.com/styled-components/styled-components/pull/683?

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ All notable changes to this project will be documented in this file. If a contri
 
 ### Changed
 
+## [v.1.4.5] - 2017-04-14
+
+### Changed
```
Comment (mxstbr) [inline CHANGELOG.md]: Forgot to update the links at the bottom!

Comment (philpl) [inline CHANGELOG.md]: Oops! Didn't see those. Updated 😆

**Comment by YasserKaddour** [comment]
@philpl Could you take a look at #692 before releasing, right now the rollup build include the `prop-types` package in it. Thanks.

**Comment by probablyup** [comment]
Could we backport the innerRef thing as well? I asked before but I don't think another 1.x release was planned at that point.

**Comment by philpl** [comment]
@jacobp100 @k15a can one of you review this? I forgot to merge it, but it's already released :)

**Pull request merged**